### DISCUSSION TITLE
Azure Push Notifications Nuget Library for MAUI iOS

### DISCUSSION THREAD

**Discussion by KSEL270715** [dotnet/maui#10327] [Q&A] [answered]
We are doing MAUI Migration. For Push notifications using [Xamarin.Azure.NotificationHubs.iO](https://www.nuget.org/packages/Xamarin.Azure.NotificationHubs.iOS/)S in our Xamarin Projects.
But unable to find Azure Notificationhub nuget library for MAUI iOS.

**Comment by aritchie** [comment, marked as answer]
I answer this here: https://github.com/dotnet/maui/issues/10330
